_topic_map.yml

```diff
@@ -531,10 +531,27 @@ Topics:
     Distros: openshift-*
   - Name: Garbage Collection
     File: garbage_collection
-    Distros: openshift-*
-  - Name: Scheduler
-    File: scheduler
+    Distros: openshift-*    
+  - Name: Advacnced Scheduling
```

Comment:
Also - I am wondering about the organization of this. This section now contains the original default scheduler and the new advanced scheduler and custom scheduler, but the overall section is called Advanced Scheduler. That is confusing. Any reason why we can't just leave it as Scheduler and include the other topics in it?